extensions/0znzw/0tils.js

```diff
@@ -0,0 +1,1565 @@
+/*!
+ * VERSION 8.6
+ * Originally created by https://scratch.mit.edu/users/0znzw/
+ * This file is available under an informal "use with credit" license.
```

Comment:
RixxyX was a one-time exception. All others need to be added under a real, proper open source license.

Reply:
k

Reply:
its MIT now

Reply:
as all the librarys and code that was not mine was licensed under MIT or [CCBY2](https://creativecommons.org/licenses/by/2.0/legalcode)